uncover/session.go

```diff
@@ -1,22 +1,25 @@
 package uncover
 
 import (
+	"context"
 	"crypto/tls"
 	"net/http"
 	"net/url"
 	"time"
 
 	"github.com/pkg/errors"
+	"github.com/projectdiscovery/ratelimit"
 	"github.com/projectdiscovery/retryablehttp-go"
 )
 
 type Session struct {
-	Keys     *Keys
-	Client   *retryablehttp.Client
-	RetryMax int
+	Keys       *Keys
+	Client     *retryablehttp.Client
+	RetryMax   int
+	RateLimits *ratelimit.MultiLimiter
```

Comment:
Users don't usually configure ratelimits to avoid unstable behaviour we should add a default ratelimit map which contains ratelimits of earch source . this data is available on source documentation. this also improves speed and makes uncover more stable